### PR TITLE
Add storybook

### PR DESCRIPTION
**Pull request by nathaliebru**

### CODE REVIEW

File: src/components/button/Button.styled.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const Button = styled.button<ButtonProps>`
   gap: 15px;
   justify-content: center;
   padding: 13px 30px;
-  width: 100%;
+  /* width: 100%; */
```
Comment (fabianovangool) [inline src/components/button/Button.styled.ts:20]: Remove commented out styling

File: src/stories/Button.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Default = Template.bind({});
+
+export default {
+  title: "Button",
+  component: Button,
+  argTypes: {
+    disabled: {
+      options: [true, false],
+      control: { type: "boolean" },
```
Comment (fabianovangool) [inline src/stories/Button.stories.tsx:15]: I think that for booleans you don't have to define the options and control type

File: src/stories/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    icon: {
+      options: [true, false],
+      control: { type: "boolean" },
```
Comment (fabianovangool) [inline src/stories/Button.stories.tsx:19]: Same as above

**Review by fabianovangool — CHANGES_REQUESTED** [review]
Looks great!

**Review by hans-acc — APPROVED** [review]
Good points by @fabianovangool! After implementing those changes I have no remarkts

**Review by fabianovangool — APPROVED** [review]

**Review by hans-acc — APPROVED** [review]

**Pull request merged**